frontend/src/pages/RollingpaperPage/components/LetterPaper.tsx

```diff
@@ -160,48 +83,41 @@ const LetterPaper = ({ to, messageList }: LetterPaperProp) => {
     <StyledLetterPaper>
       <StyledLetterPaperTop>
         <StyledTo>To. {to}</StyledTo>
-        {!writeNewMessage && !editMessageId && (
-          <IconButton size="small" onClick={handleMessageWriteButtonClick}>
+        {!isEdit && (
+          <IconButton size="small" onClick={handleWriteButtonClick}>
             <PencilIcon />
           </IconButton>
         )}
       </StyledLetterPaperTop>
       <StyledSlicedMessageLists>
         {slicedMessageLists.map((messageList, index) => (
           <StyledMessageList key={index}>
-            {index === 0 && writeNewMessage && (
+            {index === 0 && isEdit && (
               <MessageForm
-                onSubmit={submitMessageForm}
-                onCancel={cancelMessageWrite}
-                content={content}
+                onSubmit={handleMessageSubmit}
+                onCancel={handleMessageCancel}
                 onChange={handleMessageChange}
+                onClickColor={handleColorClick}
+                onClickAnonymous={handleAnonymousCheckBoxChange}
+                onClickSecret={handleSecretCheckBoxChange}
+                content={content}
                 color={color}
-                onClickColor={setColor}
+                anonymous={anonymous}
+                secret={secret}
               />
             )}
             {messageList.map((message) => {
-              if (!writeNewMessage && editMessageId === message.id) {
-                return (
-                  <MessageForm
-                    key={message.id}
-                    onSubmit={submitMessageForm}
-                    onCancel={cancelMessageWrite}
-                    content={content}
-                    onChange={handleMessageChange}
-                    color={color}
-                    onClickColor={setColor}
-                  />
-                );
-              }
               return (
                 <RollingpaperMessage
                   key={message.id}
                   content={message.content}
                   author={message.from}
                   color={message.color}
-                  authorId={message.authorId}
                   messageId={message.id}
-                  onClickEdit={handleEditMessageClick}
+                  editable={message.editable}
+                  anonymous={message.anonymous}
+                  secret={message.secret}
+                  isVisible={message.visible}
                 />
               );
```

Comment:
[여기 코멘트](https://github.com/woowacourse-teams/2022-nae-pyeon/pull/344#discussion_r945108426)에서 이어지는 이야기입니당. 
API에서는 메시지 작성자 정보를 from으로 내려주고 있는 것 같아요..!
RollingpaperMessage 컴포넌트 prop들의 이름을 수정하면 prop을 하나하나 내려주지 않아도 될 것 같다는 이야기였습니당. 

RollingpaperMessage의 prop 중 author를 from으로, isVisible이 visible로 바뀌면 아래와 같이 바로 분해 할당할 수 있을 것 같아요.
제 생각에는 prop이, AP 응답으로 돌려받은 정보와 같아도 괜찮을 것 같아서 하는 제안입니다!
```suggestion
              return (
                <RollingpaperMessage
                  key={message.id}
                  {...message}
                />
              );
```

Reply:
아하 이말이었군요. 좋습니당